cpp/tests/utilities/compare_column_wrappers.cuh

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2019 BlazingDB, Inc.
+ *     Copyright 2019 Eyal Rozenberg <eyalroz@blazingdb.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CUDF_TEST_UTILITIES_COMPARE_COLUMNS_CUH_
+#define CUDF_TEST_UTILITIES_COMPARE_COLUMNS_CUH_
+
+#include <tests/utilities/cudf_test_fixtures.h> // for GdfTest
+#include <tests/utilities/cudf_test_utils.cuh>
+#include <tests/utilities/column_wrapper.cuh>
+#include <utilities/bit_util.cuh>
+#include <utilities/type_dispatcher.hpp>
+#include <utilities/wrapper_types.hpp>
+#include <utilities/column_utils.hpp>
+
+#include <cudf.h>
+
+#include <gtest/gtest.h>
+
+#include <iostream>
+#include <iomanip>
+#include <tuple>
+
+namespace detail {
+
+template <typename T>
```

Comment:
How about instead:

```
template <typename T>
std::string to_string(T val) {
  return std::to_string(cudf::detail::unwrap(val));
}

std::string to_string(cudf::bool8 val) {
  return {cudf::detail::unwrap(val) ? "true" : "false"};
}
```

Reply:
Ok, sure.

Reply:
Ah, right, `unwrap()` is a pass-through for non-wrapped types. Sure.